src/components/util/alert-custom-message-node.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import Alert from '@mui/material/Alert';
+import { FormattedMessage } from 'react-intl';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+    customMessageNode: {
+        position: 'absolute',
+        top: '30%',
+        left: '43%',
+    },
+}));
+
+const AlertCustomMessageNode = (props) => {
+    const classes = useStyles();
+    const { noMargin, message } = props;
+
+    return (
+        <Alert
+            className={clsx({ [classes.customMessageNode]: !noMargin })}
+            severity="warning"
+        >
+            <FormattedMessage id={message} />
+        </Alert>
+    );
+};
+
+AlertCustomMessageNode.defaultProps = {
+    noMargin: false,
+    message: '',
+};
+
+AlertCustomMessageNode.propTypes = {
+    noMargin: PropTypes.bool,
+    message: PropTypes.string,
+};
+
+export default AlertCustomMessageNode;
```

Comment:
Maybe that's a better idea to reuse the AlertInvalidNode instead of creating a copy ? Add the message props to the AlertInvalidNode, and put "InvalidNode" as a default value for message ?

Reply:
To do in another PR.